packages/core-bridge/src/worker.rs

```diff
@@ -771,10 +769,7 @@ mod custom_slot_supplier {
                 slot_type: SK::kind().into(),
                 task_queue: ctx.task_queue().to_string(),
                 worker_identity: ctx.worker_identity().to_string(),
-                worker_deployment_version: ctx
-                    .worker_deployment_version()
-                    .clone()
-                    .map(Into::into),
+                worker_deployment_version: ctx.worker_deployment_version().clone().map(Into::into),
```

Comment:
Can you please check why the spurious reformat here? Could be that previous commit was not properly formatted, but that sometimes happen due to update in prettier. If the later, then we should make another PR at some point in the near future to bump our prettier dependency and reformat everything.

Reply:
my `package-lock.json` has prettier at `3.1.1` which leads me to think it's not a prettier update
(for reference, `package.json` has prettier at `^3.1.1"`